CHANGELOG.rst

```diff
@@ -1,20 +1,27 @@
 Changelog for Mantarray Frontend Components
 ===========================================
 
-0.6.2 (02-17-2022)
+0.6.3 (unreleased)
```

Comment:
do we need the state instead?

Reply:
my bad, that should have today's date, but we can probably just fix it in a future PR